routes_test.go

```diff
@@ -585,14 +585,14 @@ func TestRouteContextHoldsFullPath(t *testing.T) {
 	// Test routes
 	routes := []string{
 		"/simple",
+		"/project/:name/bui",
 		"/project/:name",
 		"/",
 		"/news/home",
 		"/news",
 		"/simple-two/one",
 		"/simple-two/one-two",
 		"/project/:name/build/*params",
-		"/project/:name/bui",
```

Comment:
put these changes back and add a new route as above.

Reply:
Done @appleboy. I added new route.